lib/ansible/modules/network/nxos/nxos_pim_rp_address.py

```diff
@@ -161,16 +119,13 @@ def apply_key_map(key_map, table):
     for key, value in table.items():
```

Comment:
`apply_key_map` can be removed from this module as this is used nowhere, but this doesn't block merging.